test/integration/targets/npm/tasks/test.yml

```diff
@@ -0,0 +1,72 @@
+- name: 'Remove any node modules'
+  file:
+    path: '{{ remote_dir }}/node_modules'
+    state: absent
+
+- vars:
+    # sample: node-v8.2.0-linux-x64.tar.xz
+    node_path: '{{ remote_dir }}/node-v{{ nodejs_version }}-{{ ansible_system|lower }}-x{{ ansible_userspace_bits }}/bin'
+  block:
+    - shell: echo $PATH
```

Comment:
Is this used for debugging, shouldn't it be removed ?

Reply:
Done